lib/libtlspool.c

```diff
@@ -37,6 +38,18 @@ static pthread_cond_t updated_poolfd = PTHREAD_COND_INITIALIZER;
 
 static pthread_mutex_t prng_lock = PTHREAD_MUTEX_INITIALIZER;
 
+int tlspool_getpid () {
+	int fd;
+
+	if ((fd = open("/var/run/tlspool.pid", O_RDONLY)) != -1) {
+		char str_pid[256];
+
+		if (read(fd, str_pid, sizeof(str_pid)) != -1) {
```

Comment:
De spontaan veranderde line endings heb ik nu gecorrigeerd. Is de pull
request nu OK?

2016-01-27 12:37 GMT+01:00 vanrein <notifications@github.com>:

> In lib/libtlspool.c
> <https://github.com/arpa2/tlspool/pull/7#discussion_r50972472>:
>
> > @@ -37,6 +38,18 @@ static pthread_cond_t updated_poolfd = PTHREAD_COND_INITIALIZER;
> >
> >  static pthread_mutex_t prng_lock = PTHREAD_MUTEX_INITIALIZER;
> >
> > +int tlspool_getpid () {
> > +	int fd;
> > +
> > +	if ((fd = open("/var/run/tlspool.pid", O_RDONLY)) != -1) {
> > +		char str_pid[256];
> > +
> > +		if (read(fd, str_pid, sizeof(str_pid)) != -1) {
>
> Hoi, Getest: 7310cea
> <https://github.com/arpa2/tlspool/commit/7310cea45c173ef9e9803c4a87fa0706e74cfee8>
> "fix 2" Hij bouwt en alles start op Linux; alleen breken testsrv en testcli
> de connectie af met "broken pipe" wat het verwachte gedrag is door de
> aanpassing van SOCK_SEQPACKET naar SOCK_STREAM Gek genoeg krijg ik ook nog
> "broken pipe" op 58efc94
> <https://github.com/arpa2/tlspool/commit/58efc944a82b632d25f7fd95b5cfa1de441d982a>
> Zoekzoek... verdraaid, die SEQPACKET is helemaal niet van belang op die
> plek, dat mag best SOCK_STREAM zijn lijkt me, want dat is het socketpair().
> Blijkbaar biedt een Pull Request toch wat weinig context om op te
> oordelen...! Dan ligt er dus nog een puzzeltje, wat er veranderd kan zijn
> aan de code...
> … <#1850943476_>
> -Rick
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/arpa2/tlspool/pull/7/files#r50972472>.
>